### ISSUE TITLE
sample_data is missing from sdist

### ISSUE THREAD

**Issue by QuLogic** [saulpw/visidata#2218]
**Small description**
When using the sdist, there is no `sample_data` directory. This causes tests to create the test data from scratch, causing them to fail as expected information isn't there.

**Expected result**
Tests pass.

**Actual result with screenshot**
Running `pytest` produces:
```
=================================== FAILURES ===================================
__________________________ TestFeatures.test_features __________________________

self = <visidata.tests.test_features.TestFeatures object at 0x7f0562954bc0>
mock_screen = <Mock id='139661071287984'>

    def test_features(self, mock_screen):
        tests = [
            (mod, getattr(mod, k))
                for mod in visidata.vd.importedModules
                    for k in dir(mod)
                        if k.startswith('test_')
        ]
        for mod, testfunc in tests:
            print(mod, testfunc.__name__)
            visidata.vd.resetVisiData()
            visidata.vd.scr = mock_screen
>           testfunc(visidata.vd)

visidata/tests/test_features.py:17:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
visidata/features/slide.py:124: in test_slide_keycol_1
    t('',                             'OrderDate Region Rep Item Units Unit_Cost Total')
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

vdx = '', golden = 'OrderDate Region Rep Item Units Unit_Cost Total'

    def t(vdx, golden):
        global vd
        vd = visidata.vd.resetVisiData()
        vd.runvdx(setup_vdx)

        vd.runvdx(vdx)
        colnames = [c.name for c in vd.sheet.visibleCols]
>       assert colnames == golden.split(), ' '.join(colnames)
E       AssertionError: H

visidata/features/slide.py:112: AssertionError
----------------------------- Captured stdout call -----------------------------
<module 'visidata.fuzzymatch' from '/builddir/build/BUILD/visidata-3.0/visidata/fuzzymatch.py'> test_fuzzymatch
<module 'visidata.features.slide' from '/builddir/build/BUILD/visidata-3.0/visidata/features/slide.py'> test_slide_keycol_1
cmdlog open-file
sample key-col
sample key-col
sample key-col
----------------------------- Captured stderr call -----------------------------
sample_data/sample.tsv does not exist, creating new sheet
sample has no column "OrderDate"
sample has no column "Region"
sample has no column "Rep"
```

**Additional context**
VisiData 3.0 and Python 3.12.

**Comment by QuLogic** [comment]
Or perhaps this test should be using `vd.pkg_resources_files(visidata) / 'tests/sample.tsv'` like I see in some other tests?

**Comment by anjakefala** [comment]
> vd.pkg_resources_files(visidata) / 'tests/sample.tsv'

Yeah, I think this is the correct approach. We should get that fixed before we update the debian. Thanks @QuLogic!

**Comment by anjakefala** [comment]
I pushed the fix! These tests will now use the packaged sample files. This change will be included in the 3.0.1 patch.
